src/services/CodebenderServiceProvider.php

```diff
@@ -0,0 +1,67 @@
+<?php
+// CodebenderServiceProvider.php
+
+namespace codebender\blocklyduino\services;
+
+use GuzzleHttp\Client;
+use Silex\Application;
+use Silex\ServiceProviderInterface;
+
+class CodebenderServiceProvider implements ServiceProviderInterface
+{
+    /**
+     * @var string The URL for the Codebender website
+     */
+    protected $codebender_url = 'https://codebender.cc';
```

Comment:
One more thing to put in your config. The codebender url is not going to change, but it's a good practice to keep such thing outside your code.